src/psyclone/transformations.py

```diff
@@ -353,6 +353,20 @@ def __init__(self, omp_schedule="static"):
         self.omp_schedule = omp_schedule
         Transformation.__init__(self)
 
+    def _validate(self, node):
+        ''' Perform validation checks before applying the transformation'''
```

Comment:
Please fully document the interface to this routine.

Reply:
OK, done.

Reply:
Err, it's not! (You've done the one above but not the `_validate` at line 372.